qemu/tests/driver_in_use.py

```diff
@@ -31,7 +31,7 @@ def check_bg_running(target_process):
                  else return False
         """
         session = vm.wait_for_login()
-        list_cmd = "wmic process get name"
+        list_cmd = params["list_cmd"]
```

Comment:
There are some cases use driver_in_use.py, if the param is a must, please update all related cases, else this change would fail other cases.